m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -3282,6 +3312,28 @@ PROCEDURE NORMALIZESLID
   END
 END [PROCEDURE]
 
```

Comment:
I'm thinking the CHARCHECKTMP variable is not necessary.  And maybe just do the SEGMENT one time to get the current character that is processing:
```
PROCEDURE CHECKFORINVALIDCHARS
[* Check the passed character value for any double-quotes (ascii 34)
** and if any found replace with a single-quote character (ascii 39).
**  PASS    CHARCHECK  Character value being tested
**  RETURN  CHARCHECKRETURN  Updated CHARCHECK character value
*]
 CHARCHECKRETURN=""

 NLTSCHR=CHARCHECK
 CALL NLTS
 CHARCHECK=NLTSCHR

 FOR CHARCHECKNUM=1 TO LENGTH(CHARCHECK)
  DO
   TMPCHR=SEGMENT(CHARCHECK,CHARCHECKNUM,CHARCHECKNUM)
   IF TMPCHR=CTRLCHR(34) THEN
    CHARCHECKRETURN=CHARCHECKRETURN+CTRLCHR(39)
   ELSE
    CHARCHECKRETURN=CHARCHECKRETURN+TMPCHR
  END
END [PROCEDURE]
```

Reply:
@rrobison44 Updated as (almost) requested